src/index.html

```diff
@@ -2,11 +2,459 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header-logo">
+        <img src="images/logo.png" class="logo"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
```

Comment:
Don't use tag name for class name, class name should be meaningful, not just contain the name of the element with this class